pkg/build/docker_generic.go

```diff
@@ -47,9 +47,6 @@ func (b *DockerGenericBuilder) Build(ctx context.Context, in *api.BuildInput, ow
 		return nil, err
 	}
 
-	ctx, cancel := context.WithTimeout(ctx, 30*time.Minute)
```

Comment:
Do we have an integration test that checks the task timeout?

Reply:
Not really, here we're dropping a 30 minutes timeout: it's a rather trivial change, and the only integration test we could write here is to wait for 30 minutes in the CI. The benefits / cost ratio would be very low.